app/assets/stylesheets/content/_tabs.sass

```diff
@@ -63,6 +63,10 @@
   li:hover
     border-bottom-color: #DDDDDD
 
+content-tabs
+  min-height: 73.5px
```

Comment:
Why this number? Half pixels normally don't exists and every browser handles it differently. As far as I could see 73px works as well. (It would be perfect, if we made the number even less magic (like 70, or 75), but I assume there are reasons against those?)

Reply:
Just looked at the browser and what was returned for the height. I'll switch it to 73, then.